modules/ipi-install-bmc-addressing.adoc

```diff
@@ -64,3 +64,62 @@ platform:
           password: <password>
           disableCertificateVerification: True
 ----
+[discrete]
+== Redfish APIs
+
+Several redfish API endpoints are called onto your BCM when using the bare-metal installer-provisioned infastructure.
+
+[IMPORTANT]
+====
+You need to ensure that your BMC supports all of the redfish APIs prior to installation.
+====
+
+**List of redfish APIs:**
+
+.PowerOn
+[source, terminal]
+----
+curl -u $USER:$PASS -X POST -H'Content-Type: application/json' -H'Accept: application/json' -d '{"Action": "Reset", "ResetType": "On"}' https://$SERVER/redfish/v1/Systems/$SystemID/Actions/ComputerSystem.Reset
+----
+
+.PowerOff
+[source, terminal]
+----
+curl -u $USER:$PASS -X POST -H'Content-Type: application/json' -H'Accept: application/json' -d '{"Action": "Reset", "ResetType": "ForceOff"}' https://$SERVER/redfish/v1/Systems/$SystemID/Actions/ComputerSystem.Reset
+----
+
+.Temporary boot
+[source, terminal]
+----
+curl -u $USER:$PASS -X PATCH -H "Content-Type: application/json"  https://$Server/redfish/v1/Systems/$SystemID/ -d '{"Boot": {"BootSourceOverrideTarget": "pxe", "BootSourceOverrideEnabled": "Once"}}
+----
+
+.BootSourceOverrideMode
+[source, terminal]
+----
+curl -u $USER:$PASS -X PATCH -H "Content-Type: application/json"  https://$Server/redfish/v1/Systems/$SystemID/ -d '{"Boot": {"BootSourceOverrideMode":"UEFI"}}
+----
+
+**List of redfish-virtualmedia APIs:**
```

Comment:
Same format suggestion in this section - maybe make this title a `.Heading` and the items description lists